docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -683,6 +683,53 @@ Where adherence score = 0 for primary or secondary nonadherent; and adherence sc
     - Burn in period will allow the distribution of follow-up appointments to reach equilibrium prior to time start 
 
 
+Creation of "Untreated" SBP Values on Initializaiton
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GBD values for SBP which are used in this sim reflect the US distribution of SBP **including** medication benefits. 
+Therefore, by later applying treatment benefits to certain simulants, we are double counting the population level 
+benefit of treatment. To avoid this, we must add SBP at initialization to create an "untreated" SBP level. 
+
+To do this, we will add a percent increase in SBP to all simulants who are assigned SBP medication at 
+initialization. The section below includes details on who will receive medication. 
+
+In itialization simulants are also assigned to one drug or two drugs. Based on this assignment, a percent increase 
+in SBP level will be given to each. 
+
+.. list-table:: Percent Increase in SBP 
+  :widths: 10 10 
+  :header-rows: 1
+
+  * - Medication Assigned 
+    - Percent Decrease 
+  * - One Drug  
+    - 5.1% 
+  * - Two Drugs  
+    - 12%  
```

Comment:
I think it would be good to add references here! and in the table below as well. I also agree with Abie that there should be more detail on how to do the "baseline calibration." Some documented examples include the [zinc supplementation](https://vivarium-research.readthedocs.io/en/latest/intervention_models/zinc_supplementation/index.html) and [antenatal supplementation](https://vivarium-research.readthedocs.io/en/latest/intervention_models/maternal_supplementation/index.html) interventions. Let me know if you'd like to chat about it at all!

Reply:
Just noting that Syl and I met to discuss and she filled me in that this modeling strategy will take a different approach than the typical [baseline calibration](https://vivarium-research.readthedocs.io/en/latest/model_design/concept_model/baseline_coverage/index.html) approach because the barrier to doing so is high and the limitations of doing so are acceptable for the project and will add some more explanation to the docs to make this more clear!

Reply:
Nit: it would be nice to have the reference actually in the table, so you can see at a glance which source each number came from.